project.html

```diff
@@ -4,38 +4,38 @@
 permalink: project
 ---
 
-<p><strong>KeePassXC</strong> is a modern, secure, and open-source password manager that stores and manages your most sensitive information.</p>
+<p><strong>KeePassXC</strong> is a modern, secure, and open-source password manager that stores and manages your most sensitive info.</p>
```

Comment:
Why is info better here than information? It's the same, just more informal and less precise.

Reply:
It is shorter. Just as precise.